### PR TITLE
[INTEL MKL] Update build name with new branding

### PR DESCRIPTION
**Pull request by gaurides**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,8 +133,8 @@ Build Type                                                        | Status
...
-**Linux CPU with Intel® MKL-DNN** Nightly                         | [![Build Status](https://tensorflow-ci.intel.com/job/tensorflow-mkl-build-whl-nightly/badge/icon)](https://tensorflow-ci.intel.com/job/tensorflow-mkl-build-whl-nightly/)                     | [Nightly](https://tensorflow-ci.intel.com/job/tensorflow-mkl-build-whl-nightly/)
-**Linux CPU with Intel® MKL-DNN** Stable Release                  | ![Build Status](https://tensorflow-ci.intel.com/job/tensorflow-mkl-build-release-whl/badge/icon)                                                                                              | Release [1.15](https://pypi.org/project/intel-tensorflow/1.15.0/) / [2.x](https://pypi.org/project/intel-tensorflow/)
+**Linux Intel CPU** Nightly <br> oneAPI Deep Neural Network Library (oneDNN)        | [![Build Status](https://tensorflow-ci.intel.com/job/tensorflow-mkl-build-whl-nightly/badge/icon)](https://tensorflow-ci.intel.com/job/tensorflow-mkl-build-whl-nightly/)                     | [Nightly](https://tensorflow-ci.intel.com/job/tensorflow-mkl-build-whl-nightly/)
```
Comment (penpornk) [inline README.md:136]: This build actually works with x86/x86-64 CPUs, not just Intel. Also, oneDNN should be in the build name (bold part) to make it clearer how it's different from vanilla TensorFlow build. How about:
**Linux CPU with Intel oneAPI Deep Neural Network Library (oneDNN)** Nightly
or
**Linux x86 CPU with Intel oneAPI Deep Neural Network Library (oneDNN)** Nightly
?

**Review by penpornk — COMMENTED** [review]
Thank you for the PR!

File: README.md (hunk 1 above)
Comment (gaurides) [reply README.md]: Thanks for the feedback. I submitted a newer commit with first suggestion.

**Review by mihaimaruseac — APPROVED** [review]

**Review by penpornk — APPROVED** [review]
Thank you for the changes!

**Pull request merged**